tests/integration/base.py

```diff
@@ -443,7 +443,7 @@ def create_devices(klass):
         klass.dev_w_client = devs[1]
 
         # don't let NM trample over our fake AP
-        with open('/run/NetworkManager/conf.d/test-blacklist.conf', 'w') as f:
+        with open('/run/NetworkManager/conf.d/test-blocklist.conf', 'w') as f:
```

Comment:
nitpick: I'd prefer denylist instead of blocklist (also above).

Reply:
There are more instances of the `test-blocklist.conf` below in `tearDownClass()`.